app/services/subcase-is-approved.js

```diff
@@ -13,7 +13,7 @@ export default class SubcaseIsApprovedService extends Service {
       sort: '-planned-start',
     });
 
-    if (latestMeeting?.isFinal) {
+    if (latestMeeting?.agenda) {
```

Comment:
Similar comment as on `app/controllers/agenda/agendaitems/agendaitem.js`. `latestMeeting?.agenda` returns a promise. You need to explicitely `await` the relation and check on the resolved value.